internal/k8shandler/index_management.go

```diff
@@ -42,10 +42,19 @@ func (er *ElasticsearchRequest) CreateOrUpdateIndexManagement() error {
 		return err
 	}
 	primaryShards := getDataCount(er.cluster)
+
+	suspend := true
+	for _, node := range cluster.Spec.Nodes {
+		if node.NodeCount > 0 {
+			suspend = false
+			break
+		}
+	}
```

Comment:
this doesn't change/solve the use case as mentioned in the jira. the operator does not change the number of nodes in the spec at all

Reply:
You say we should account also for `len(cluster.Spec.Nodes) == 0`?

Reply:
no, because the operator doesn't change this at all.
We should instead be checking if there are any ES pods out there running -- and we have functions already that do this for us.

Reply:
Issue addressed by listing if any es pods exist.